### ISSUE TITLE
Licence Missing

### ISSUE THREAD

**Issue by ghost** [alisan16/icorruption-ext#2]
Hello! I think you should add a COPYING.txt (or Licence.txt) file in your repository.

**Comment by jeromecc** [comment]
License. ;-)

**Comment by stemarco** [comment]
Good point, I'll check with Chris to see if there is a preference, but it is open source
